N/Nabla/Compat.toml

```diff
@@ -14,13 +14,20 @@ julia = "0.6-1.3"
 ["0.1.1-0.10"]
 FDM = "0.1-0.4"
 
-["0.11"]
+["0.11.0"]
 DiffRules = "0.0.0-0.0"
 DualNumbers = "0.6.0-0.6"
 FDM = ["0.1", "0.2"]
 SpecialFunctions = "0.5.0-*"
 julia = "1.0.0-1"
 
+["0.11.1"]
+DiffRules = "0.0"
+DualNumbers = "0.6"
+FDM = "0.1-0.4"
+SpecialFunctions = "0.5-0.7"
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.